src/backend/mailbox_manager/models.py

```diff
@@ -23,6 +24,13 @@ class MailDomain(BaseModel):
     name = models.CharField(
         _("name"), max_length=150, null=False, blank=False, unique=True
     )
+    organization = models.ForeignKey(
+        Organization,
+        on_delete=models.PROTECT,
+        related_name="mail_domains",
+        null=True,
```

Comment:
Is the plan to later switch this to null=False ?

Reply:
A domain doesn't really require an Organization today, it's only mandatory for ProConnect for now... But when we will improve permissions, it may become:

- easy to determine the organization, or at least propose a set of organization to the end user
- mandatory to have an organization to be able to determine who can manage mailboxes

So yeah, it may become null=False in a near future :) While the "near future" is not clear, I did not mention it in the code ^^'